overviewer_core/textures.py

```diff
@@ -975,16 +975,18 @@ def saplings(self, blockid, data):
     # usual saplings
     tex = self.load_image_texture("assets/minecraft/textures/block/oak_sapling.png")
     
-    if data & 0x3 == 1: # spruce sapling
+    if data == 1: # spruce sapling
```

Comment:
pretty sure that `& 0x3` was there because pre-1.13 Minecraft stored growth age in the block data. I propose we investigate how acacia saplings were stored in 1.12, and then fix the function for that, and then use separate IDs for the new ones and the `sprite` shorthand for them instead of using the data field and having a needlessly complicated material definition.

Reply:
I did some quick poking about and the current master does not even work below 1.13:
![afbeelding](https://user-images.githubusercontent.com/8080706/63180446-99184f00-c04e-11e9-95a6-3f372c80d9b9.png)
![afbeelding](https://user-images.githubusercontent.com/8080706/63180466-a1708a00-c04e-11e9-8661-494878c29a73.png)

All the sapling blocks pre-1.13 are just called `minecraft:sapling` with a `type` tag. And to be frank I don't see why that one oak sapling actual rendered since I can't find a reference to `minecraft:sapling`. (I guess it still works via an id in this version? Is has been too long 😅)

So I guess the best option would be create separate sapling blocks with their own ID and add an issue for sapling backwards compatibility?

Reply:
>So I guess the best option would be create separate sapling blocks with their own ID and add an issue for sapling backwards compatibility?

Probably, yeah.